### DISCUSSION TITLE
"Tab blocks" /  Horizontal block editor

### DISCUSSION THREAD

**Discussion by DennisSnijder** [payloadcms/payload#2552] [Feature Requests & Ideas]
Whenever working with [Blocks](https://payloadcms.com/docs/fields/blocks), you're always adding new blocks on the vertical axis. It would be nice to have blocks on the horizontal axis (Tabs, but editable) for adding / managing blocks. 

Here's a mockup of what something like this would look like:
<img width="1162" alt="Screenshot 2023-04-24 at 21 48 46" src="https://user-images.githubusercontent.com/6953846/234100731-92fd0377-17fa-463d-922d-2d4be3c03972.png">

**Comment by tylandavis** [comment]
I would love a feature like this. Would be nice to have this for both Blocks and Arrays.
